pkg/logs/tailers/windowsevent/tailer.go

```diff
@@ -46,6 +46,46 @@ const (
 type Config struct {
 	ChannelPath string
 	Query       string
+	// V1ProcessingBehavior indicates ifq we want to process and send the whole structured log message
+	// instead of on the logs content. Note that the default behavior is now to only send
+	// the logs content, as other tailers do.
+	V1ProcessingBehavior bool
+}
+
+// eventContext links go and c
+type eventContext struct {
```

Comment:
Where `eventContext` is used? I cannot find any reference to it

Reply:
I'll look into this, it might be an artifact from me catching-up with the `main` branch.